src/ol-ext/interaction/measure.js

```diff
@@ -250,6 +256,16 @@ ngeo.interaction.Measure.prototype.onDrawStart_ = function(evt) {
   this.sketchFeature = evt.feature;
   this.overlay_.getFeatures().clear();
   this.createMeasureTooltip_();
+
+  this.changeEventKey = this.sketchFeature.getGeometry().on('change',
```

Comment:
I tried to do the following
```
goog.events.listen(geometry, goog.events.EventType.CHANGE, ...
```
but I get the following error when compiling:
```
ERR! compile /home/pierre/public_html/ngeo/src/ol-ext/interaction/measure.js:263: ERROR - actual parameter 1 of goog.events.listen does not match formal parameter
ERR! compile found   : (null|ol.geom.Geometry|undefined)
ERR! compile required: (EventTarget|goog.events.Listenable|null)
ERR! compile   this.changeEventKey_ = goog.events.listen(geometry,
ERR! compile                                             ^
ERR! compile 
ERR! compile 
ERR! compile 1 error(s), 0 warning(s), 98.0% typed
ERR! compile 
ERR! Process exited with non-zero status, see log for more detail: 1
```

Doesn't `ol.geom.Geometry` inherit from `goog.events.Listenable`?